internal/config/conmonmgr/conmonmgr.go

```diff
@@ -0,0 +1,107 @@
+package conmonmgr
+
+import (
+	"path"
+	"regexp"
+	"strconv"
+
+	"github.com/cri-o/cri-o/utils/cmdrunner"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	majorVersionSupportsSync = 2
+	minorVersionSupportsSync = 0
+	patchVersionSupportsSync = 19
```

Comment:
Hm, we probably can use a semver.Version here. Not sure about splitting up the version into three consts.